### DISCUSSION TITLE
Request for sound/midi restart button

### DISCUSSION THREAD

**Discussion by thbeckpr** [GrandOrgue/grandorgue#1156] [Developers] [answered]
For various reasons I often have to restart the sound/midi engine in order to reestablish links to my MIDI devices. It is frustrating to have to go through File/Settings every time. I

Is there a possibility of putting a button on the main panel which would accomplish the restart/reset?

**Comment by oleg68** [comment, marked as answer]
The button is already in the main panel. It is called `Panic`.

![Panic](https://user-images.githubusercontent.com/11586506/184189299-66c48adb-c670-4af1-b490-585b5e0127d9.png)

Another way is `Audio/MIDI`->`Panic`

**Reply by thbeckpr** [reply]
Thank you. I didn't realize that is how the Panic button worked.
